schema/data/TerrasosProjectInfoData.yaml

```diff
@@ -0,0 +1,78 @@
+name: "Amazon Rainforest Conservation Project"
+description: "This project aims to conserve the Amazon Rainforest by implementing sustainable land management practices."
+location: 
+  bbox:
+    - -75.0
+    - -5.0
+    - -70.0
+    - 0.0
+  center:
+    - -72.5
+    - -2.5
+  type: Feature
+  geometry:
+    type: Point
+    coordinates:
+      - -72.5
+      - -2.5
+marketType: COMPLIANCE_MARKET
+bioregion: "Amazon Basin"
+biomeType: "TropicalForest"
+watershed: "Amazon River"
+subWatershed: "Upper Amazon"
+environmentType: CLOUD_FOREST
+offchainCreditsInfo:
```

Comment:
@wgardiner was the consensus we wanted to leave these? Or remove? I think we could remove these as it's a dynamic field they might want to change so it would make more sense to do this in sanity.

Reply:
yes we can drop `offchainCreditsInfo` since we want it to be stored in sanity

Reply:
Yes, we're going to remove `offchainCreditsInfo` and manage it with sanity. I was planning to do that in my other PR that's still open after this one gets merged, but now I'm thinking it makes more sense to merge that PR first and rebase this one on top of it.

Reply:
I've removed these